test/languages/nim_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+
+module SnippetExtractor
+  module Languages
+    class NimTest < Minitest::Test
+      def test_full_example
+        code = <<~CODE
+          # This is a file
+          # With some comments in it
+
+          # And a blank line ⬆️
+          # It has some module-related code like this:
+          import json
+          export json
+          from json import nil
+          # And then eventually the code
+          proc twoFer =
+            ##[ doc comment ]##
+            ## more doc comment
+            discard
```

Comment:
```suggestion
            # A non-doc comment
            discard
```

Reply:
What is the purpose of this change?

Reply:
No big deal - I just think it clarifies that the extracting implementation doesn't have special handling of doc comments, and regular comments are also included if they come after the first "interesting" line.

Looks like you've added it - marking as resolved.

Reply:
Ya, if you look at the [second last commit](https://github.com/exercism/generic-snippet-extractor/pull/13/commits/af62ec7a70f8d15e8aedb0803448a4e58e14d3e8), I added a regular comment after the proc that isn't excluded. The next commit also adds a doc comment as the first line in the test file.